packages/app/src/components/actions/deposit.tsx

```diff
@@ -40,45 +40,64 @@ const Deposit: FC<Props> = ({ name, symbol, tokenBalance, tokenAllowance }) => {
   const isTokenApproved = useIsTokenApproved(addresses[name].token, addresses[name].controller, depositInput);
 
   return (
-    <Form>
-      <Form.Group controlId="formDeposit">
-        <Form.Label>Deposit Amount</Form.Label>
-        <Input
-          width={1}
-          value={depositInput}
-          onChange={handleInputChange}
-          type="text"
-          placeholder={"0 " + symbol}
-          className="amount"
-        />
-        <Form.Text className="balance" onClick={maxDeposit}>
-          Current Balance: {`${utils.formatEther(tokenBalance?.toString() || "0")} ${symbol}`}
-        </Form.Text>
-      </Form.Group>
-      <div className="d-grid gap-2">
-        <ApproveToken
-          symbol={symbol}
-          spender={addresses[name].controller}
-          token={contracts[name].token}
-          hasAllowance={isTokenApproved}
-        />
-        <Button
-          disabled={
-            !isTokenApproved || !depositInput || depositInput.toString() === "0" || depositTx.status === "Mining"
-          }
-          onClick={depositTokens}
-        >
-          {depositTx.status === "Mining" ? (
-            <>
-              <Spinner animation="border" variant="white" />
-              Depositing...
-            </>
-          ) : (
-            "Deposit"
-          )}
-        </Button>
-      </div>
-    </Form>
+    <>
```

Comment:
this is not needed here. I also happened to shoot myself in the foot yesterday with this, because if there are some conditional rendering inside a Fragment, it can mess up `grid` layout